tests/PilotLive/Models/BaseModelTest.php

```diff
@@ -132,11 +124,9 @@ protected function verifyGetUndefined(string $class)
     protected function verifyCanNotNullify(string $class, string $key)
     {
         $className = $this->getClassName($class);
-
         $this->expectException(ModelException::class);
         $this->expectExceptionMessage("Model \"{$className}\" attempting to nullify key {$key} Property is null without nullable permission");
         $this->expectExceptionCode(10111);
-
         $model = new $class($this->config);
         $model->{$key} = null;
```

Comment:
I think the previous way with the 3 distinct segments, (1) we get the class, (2) we set expectations, (3) we so the thing, is the way to leave it.

Code must tell a story, and stories have paragraphs 😺